src/helpers/messages.js

```diff
@@ -40,7 +40,7 @@ function getExitMessage(message, scope, runner, sync) {
         itemsCountMessage = getUnansweredItemsWarning(scope, runner, sync);
     }
 
-    return `${itemsCountMessage} ${message}`.trim();
+    return message ? message : itemsCountMessage.trim();
```

Comment:
we discussed this part, should be ` return `${itemsCountMessage} ${message}`.trim();`